chrome/content/zotero/xpcom/data/item.js

```diff
@@ -4181,6 +4181,7 @@ Zotero.Item.prototype.toJSON = Zotero.Promise.coroutine(function* (options) {
 	
 	obj.dateAdded = Zotero.Date.sqlToISO8601(this.dateAdded);
 	obj.dateModified = Zotero.Date.sqlToISO8601(this.dateModified);
+	if(obj.accessDate) obj.accessDate = Zotero.Date.dateToISO(Zotero.Date.sqlToDate(obj.accessDate));
```

Comment:
No, it's stored as UTC — we don't store anything as local time — and returned from getField as UTC as well. So I think this can just use sqlToISO8601.